pages/docs/index.js

```diff
@@ -249,14 +249,14 @@ export default function DocsIndex() {
         <p>
           Here you should find everything you need from getting started with
           creating your Profile to more advanced topics. You can contribute to
-          our documentation and find the files here{" "}
+          our documentation and find the files{" "}
           <a
             href="https://github.com/EddieHubCommunity/LinkFree/tree/main/pages/docs"
             target="_blank"
             rel="noreferrer"
             className="text-sky-700 break-words"
           >
-            https://github.com/EddieHubCommunity/LinkFree/tree/main/pages/docs
+            here
```

Comment:
Using `here` is not descriptive enough for the link. It would be better to have something like this...

```html
our documentation and find the <a>markdown files</a>
```